[FIX] website: fix `s_opening_hours` background image

xlu-odoo opened the pull request:
This commit addresses a usability issue in the s_opening_hours snippet.

Previously, the background image was applied through a grid item, which made it difficult to edit the image, adjust the snippet's height or padding, and even delete the snippet itself.

With this fix, the background image is now applied directly to the section as a background image, and adjustments have been made to grid items. These changes improve the ease of editing and enhance the overall user experience.

task-4207030
Part-of: task-4077427

requires: https://github.com/odoo/design-themes/pull/1004

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/181170.png)](https://mergebot.odoo.com/odoo/odoo/pull/181170)

xlu-odoo commented:
@odoo/rd-website can you please check this PR and eventually merge?

xlu-odoo commented:
@odoo/rd-website reminder

qsm-odoo reviewed (changes requested):
Using a background-image instead seems nice but I think you may want to review the mobile mode with the default image, it's not perfect. Maybe it's just about using a new image? Don't know.

xlu-odoo commented:
> Using a background-image instead seems nice but I think you may want to review the mobile mode with the default image, it's not perfect. Maybe it's just about using a new image? Don't know.

Hello Quentin 👋 , thanks for reviewing this one.

You're right—it didn’t look great on mobile. I’ve updated it with a different image (popup snippet image fits well imo - default and others themes). I also had to adjust one of the themes (artists) that had a customization. Removed an XPath. (new related PR here: https://github.com/odoo/design-themes/pull/1004 )

lebl-odoo commented:
Not sure how you get the first screeshot, but I agree the mobile version is quite tricky. I'm expecting a result like the following: 
![image](https://github.com/user-attachments/assets/c57cb2e2-f0e4-4ec0-ac5e-37f50d195ca3)

lebl-odoo commented:
Hi @qsm-odoo , @xlu-odoo :wave: 

Except @xlu-odoo has another solution to propose, I would pick the one that suits the most the user which is the current solution (from the PR). I think basic users will use the snippet as is. For more advanced users, if the snippet doesn't meet their expectations, they are able to rebuild it from scratch.

It's not the first snippet with limitations we have and I prefer simplicity (with constraints) to complexity.

qsm-odoo commented:
Alright, then let's go with your solution for now @xlu-odoo 

Still the

> Also, I am not really convinced by using the popup default image [...]

part to handle IMO :pray:

xlu-odoo commented:
> Alright, then let's go with your solution for now @xlu-odoo
> 
> Still the
> 
> > Also, I am not really convinced by using the popup default image [...]
> 
> part to handle IMO 🙏

Hello Quentin ( @qsm-odoo ),
Reviving this one. Rebased + created new image record for `s_opening_hours` (instead of using popup_default_image)

![image](https://github.com/user-attachments/assets/f407f9d0-8e22-4162-83a0-94baf6d0e2ad)

qsm-odoo reviewed (approved):
That's probably nice :crossed_fingers: 

@robodoo r+

robodoo commented:
@xlu-odoo @qsm-odoo linked pull request(s) odoo/design-themes#1004 not ready. Linked PRs are not staged until all of them are ready.

robodoo commented:
@xlu-odoo @qsm-odoo unable to stage: merge conflict

qsm-odoo reviewed (approved):
@robodoo r+
